wfl/Database/MongoDBManager.cs

```diff
@@ -32,5 +32,32 @@ public MongoDBManager()
             IAsyncCursor<RestaurantDto> task = await collection.FindAsync(r => true, null);
             return task.ToEnumerable<RestaurantDto>();
         }
+
+        public async Task<IEnumerable<FoodOrderDto>> GetAllFoodOrdersAsync()
+        {
+            IMongoCollection<FoodOrderDto> collection = db.GetCollection<FoodOrderDto>("food-orders");
+            IAsyncCursor<FoodOrderDto> task = await collection.FindAsync(order => true, null);
+            return task.ToEnumerable<FoodOrderDto>();
+        }
+
+        public async Task InsertFoodOrderAsync(FoodOrderDto order)
+        {
+            order.Id = MongoDB.Bson.ObjectId.GenerateNewId().ToString();
+            IMongoCollection<FoodOrderDto> collection = db.GetCollection<FoodOrderDto>("food-orders");
+            await collection.InsertOneAsync(order);
+        }
+
+        public async Task<bool> CheckRestaurantIdAsync(string id)
```

Comment:
Not part of this PR, but I think we'll spend some time talking about design patterns we want to use moving forward.  There is going to start being a lot of methods in this class.